zcash_test_vectors/transaction.py

```diff
@@ -385,6 +412,11 @@ def __init__(self, rand, consensus_branch_id):
         if have_transparent_in:
             for _ in range((rand.u8() % 3) + 1):
                 self.vin.append(TxIn(rand))
+        if is_coinbase:
+            self.vin.append(TxIn.from_components(
+                OutPoint.from_components(b'\x00' * 32, 0xFFFFFFFF),
+                Script.from_bytes(b"\x51"),
+                0))
```

Comment:
The default `nSequence` is `std::numeric_limits<unsigned int>::max()`, and `src/miner.cpp` doesn't alter it.
```suggestion
                0xffffffff))
```